include/inexor/vulkan-renderer/wrapper/instance.hpp

```diff
@@ -53,6 +50,16 @@ class Instance {
     Instance &operator=(const Instance &) = delete;
     Instance &operator=(Instance &&) = default;
 
+    /// @brief Check if a certain instance layer is available on the system.
+    /// @param layer_name The name of the instance layer.
+    /// @return ``true`` if the instance layer is supported.
+    [[nodiscard]] static bool instance_layer_supported(const std::string &layer_name);
```

Comment:
Name is a little long. Since it's already in the `Instance` class, maybe it could just be called `supports_layer` or something? Same goes for the device functions too.

Reply:
or `is_layer_supported` i prefer the `is_` at the beginning

Reply:
Done.